torchao/sparsity/README.md

```diff
@@ -37,7 +37,6 @@ On Meta LLama3, we observe a 25% tok/s increase (180 -> 226) compared to our exi
 | Model       | Technique               | Tokens/Second | Memory Bandwidth (GB/s) | Peak Memory (GB) | Model Size (GB) |
 | ----------- | ----------------------- | ------------- | ----------------------- | ---------------- | --------------- |
 | Llama-3-8B  | Base (bfloat16)         |   95.64       | 1435.54                 | 16.43            | 15.01           |
-|             | int8dq                  |    8.61       |   64.75                 |  9.24            |  7.52           |
```

Comment:
do you want to remove https://github.com/pytorch/ao/blob/e283743b3cc4612bb641b88dca3670231724d396/torchao/_models/sam/benchmark.sh#L3-L4 as well?

Reply:
I think this is for LLaMa only, not for sam.